src/lib.rs

```diff
@@ -60,6 +61,7 @@ impl Plugin for NetworkingPlugin {
         app.insert_resource(NetworkResource::new(
             task_pool,
             self.link_conditioner.clone(),
+            self.message_flushing_strategy.clone(),
```

Comment:
Is `clone` needed here?

Reply:
Nope, i will make MessageFlushingStrategy derive Copy and remove the clone

Reply:
The semantics are bit different. Copy means that type is trivially copyable via bitblit, Clone means that making another entity (clone) might need some work (i.e. duplicating a file descriptor).